test/views/employment/CheckEmploymentDetailsViewSpec.scala

```diff
@@ -399,21 +399,28 @@ class CheckEmploymentDetailsViewSpec extends ViewUnitTest {
         linkCheck(s"${common.changeLinkExpected} ${common.changeEmployerNameHiddenText}", cyaChangeLink(1), EmployerNameController.show(taxYearEOY, employmentId).url)
         textOnPageCheck(common.payeReferenceField2, summaryListRowFieldNameSelector(2))
         textOnPageCheck(common.notProvided, summaryListRowFieldAmountSelector(2), "paye ref")
-        linkCheck(s"${common.addLinkExpected} ${specific.changePAYERefHiddenText}", cyaChangeLink(2), PayeRefController.show(taxYearEOY, employmentId).url)
+
+        val (payeHiddenText, employmentStartDateHiddenText) = if (userScenario.isWelsh) {
```

Comment:
This should not be done here. You should be using the content from the Specific and Common ExpectedResults.